cfgov/unprocessed/css/organisms/info-unit-group.less

```diff
@@ -0,0 +1,63 @@
+/* topdoc
+  name: Info Unit Group
+  family: cfgov-organisms
+  patterns:
+    - name: Single column (old 25/75)
+      markup: |
+        <div class="o-info-unit-group">
+            <h2>Group heading</h2>
+            <div class="content-l">
+                <div class="content-l_col content-l_col-1">
+                    info-unit
+                </div>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-info-unit-group
+            h2
+            .content-l
+              .content-l_col.content-l_col-1
+                .info-unit
+    - name: Two column
+      markup: |
+        <div class="o-info-unit-group">
+            <h2>Group heading</h2>
+            <div class="content-l">
+                <div class="content-l_col content-l_col-1-2">
+                    info-unit
+                </div>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-info-unit-group
+            h2
+            .content-l
+              .content-l_col.content-l_col-1-2
+                .info-unit
+  tags:
+    - cfgov-organisms
+*/
+.o-info-unit-group {
+    .respond-to-min(@bp-sm-min, {
+        .content-l_col-1-2:nth-of-type(n+3) {
+            // Adds margin top to info units that are 3rd and up in a group
+            margin-top: unit(@grid_gutter-width / @base-font-size-px, em);
+        }
+
+        .content-l_col-1 + .content-l_col-1 {
+            // Adds margin top to info units that follow another
+            margin-top: unit( @grid_gutter-width / @base-font-size-px, em ) !important;
```

Comment:
You've probably looked at this already, but any way we can eliminate the `!important`?